rasa/core/policies/ensemble.py

```diff
@@ -69,6 +70,15 @@ def __init__(
         self._check_priorities()
         self._check_for_important_policies()
 
+        self._rule_only_slots, self._rule_only_loops = self._get_rule_only_slots_loops()
```

Comment:
out of the scope of this PR: In my opinion all the content of this `__init__` should go to the constructor of `SimplePolicyEnsemble`. In my opinion there is too much specific behavior in the abstract interface.